Example Project/ViewController.m

```diff
@@ -137,4 +142,22 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
     NSLog(@"End displaying cell at %@", indexPath);
 }
 
+- (BOOL)flowLayoutUsedInSection:(NSInteger)section
+{
+    return section > 3 && section % 2 == 0;
```

Comment:
Just for making the actual scrolling easier to test, please only use flow layout in a single section of the example project!